sdk/python/kfp/dsl/component_factory.py

```diff
@@ -69,22 +70,34 @@ def _python_function_name_to_component_name(name):
     return name_with_spaces[0].upper() + name_with_spaces[1:]
 
 
-def make_index_url_options(pip_index_urls: Optional[List[str]]) -> str:
+def make_index_url_options(pip_index_urls: Optional[List[str]],
+                           pip_trusted_hosts: Optional[List[str]]) -> str:
     """Generates index url options for pip install command based on provided
```

Comment:
The original docstring is  strange to me, can we fix this as part of this pr? 


For example convert: 

```txt
        - Empty string if pip_index_urls is empty/None.
        - '--index-url url --trusted-host url ' if pip_index_urls contains 1
        url
        - the above followed by '--extra-index-url url --trusted-host url '
        for
        each next url in pip_index_urls if pip_index_urls contains more than 1
        url
```

to: 

```
        - Empty string if pip_index_urls is empty/None.
        - '--index-url url --trusted-host url ' if pip_index_urls contains 1 url
        - followed by '--extra-index-url url --trusted-host url ' for each next url in pip_index_urls if pip_index_urls contains more than 1 url
```

Reply:
Fixed